src/Microsoft.Identity.Web/GlobalSuppressions.cs

```diff
@@ -0,0 +1,137 @@
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:Prefix local calls with this", Justification = "not applicable", Scope = "member", Target = "~M:Microsoft.Identity.Web.TokenCacheProviders.Session.MsalSessionTokenCacheProvider.ReadCacheBytesAsync(System.String)~System.Threading.Tasks.Task{System.Byte[]}")]
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:Prefix local calls with this", Justification = "not applicable", Scope = "member", Target = "~M:Microsoft.Identity.Web.TokenCacheProviders.Session.MsalSessionTokenCacheProvider.RemoveKeyAsync(System.String)~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:Prefix local calls with this", Justification = "not applicable", Scope = "member", Target = "~M:Microsoft.Identity.Web.TokenCacheProviders.Session.MsalSessionTokenCacheProvider.WriteCacheBytesAsync(System.String,System.Byte[])~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:Prefix local calls with this", Justification = "not applicable", Scope = "member", Target = "~P:Microsoft.Identity.Web.MicrosoftIdentityOptions.DefaultUserFlow")]
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:Prefix local calls with this", Justification = "not applicable", Scope = "member", Target = "~P:Microsoft.Identity.Web.MicrosoftIdentityOptions.IsB2C")]
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:Prefix local calls with this", Justification = "not applicable", Scope = "member", Target = "~P:Microsoft.Identity.Web.TokenAcquisition.CurrentHttpContext")]
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:Prefix local calls with this", Justification = "not applicable", Scope = "member", Target = "~P:Microsoft.Identity.Web.TokenCacheProviders.Session.MsalSessionTokenCacheProvider.CurrentHttpContext")]
+[assembly: SuppressMessage("Globalization", "CA1303:Do not pass literals as localized parameters", Justification = "not applicable", Scope = "member", Target = "~M:Microsoft.Identity.Web.Base64UrlHelpers.DecodeToBytes(System.String)~System.Byte[]")]
+[assembly: SuppressMessage("StyleCop.CSharp.OrderingRules", "SA1201:Elements should appear in the correct order", Justification = "not applicable", Scope = "member", Target = "~F:Microsoft.Identity.Web.Resource.OpenIdConnectMiddlewareDiagnostics.s_onRedirectToIdentityProvider")]
+[assembly: SuppressMessage("StyleCop.CSharp.NamingRules", "SA1308:Variable names should not be prefixed", Justification = "not applicable", Scope = "member", Target = "~F:Microsoft.Identity.Web.Resource.OpenIdConnectMiddlewareDiagnostics.s_onRedirectToIdentityProvider")]
```

Comment:
Hmm. Should discuss [SA1101](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1101.md), [SA1201](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md), [SA1308](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1308.md), [CA1303](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1303?view=vs-2019).